### PR TITLE
redux-toolkit / storybook / i18n changes / switch button

### PR DESCRIPTION
**Pull request by denisstep99**

### CODE REVIEW

File: src/store/notes/Slice.ts [hunk 1]
```diff
@@ -0,0 +1,82 @@
+import {createSlice, PayloadAction, CaseReducer} from '@reduxjs/toolkit'
```
Comment (denisstep99) [inline src/store/notes/Slice.ts:1]: slice

File: src/store/notes/Types.ts [hunk 2]
```diff
@@ -1,26 +1,10 @@
-import {ActionCustomType} from "../utils";
```
Comment (denisstep99) [inline src/store/notes/Types.ts:1]: types

File: src/sagas/notes/Types.ts [hunk 3]
```diff
@@ -1,4 +1,5 @@
 import {INote} from "../../store/notes/Types";
```
Comment (denisstep99) [inline src/sagas/notes/Types.ts:1]: saga types

File: src/sagas/notes/Actions.ts [hunk 4]
```diff
@@ -6,47 +6,10 @@ import {
     RemoveNoteRequestAction,
```
Comment (denisstep99) [inline src/sagas/notes/Actions.ts:6]: saga actions

**Review by Public13 — APPROVED** [review]
this is horosho

**Pull request merged**